domain/src/main/java/org/oppia/android/domain/topic/StoryProgressTestHelper.kt

```diff
@@ -232,6 +233,50 @@ class StoryProgressTestHelper @Inject constructor(
     )
   }
 
+  /** Marks full story progress for a particular profile. */
+  fun markFullProgressForFirstTopics(profileId: ProfileId, timestampOlderThanAWeek: Boolean) {
```

Comment:
Please add tests in `StoryProgressTestHelperTest` for these new functions.

Reply:
This function is not used anywhere so removed it.